### PR TITLE
[-] Technical - Change ESLint ruleset for Airbnb

### PR DESCRIPTION
**Pull request by ArnaudValensi**
## Pull Request checklist:

- [ ] Write an explicit title for the Pull Request
- [ ] Write changes made in the CHANGELOG.md
- [ ] Test manually the implemented changes
- [ ] Review my own code (indentation, syntax, style, simplicity, readability)
- [ ] Wonder if you can improve the existing code

### CODE REVIEW

File: .eslint-bin/js-list.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+  ".eslint-bin/lint-regitered-files.js",
+  ".eslint-bin/pre-commit-hook.js"
```
Comment (arnaudbesnier) [inline .eslint-bin/js-list.json]: Needs a changelog description ;)

**Comment by ArnaudValensi** [comment]
[Express - Add airbnb coding style](https://trello.com/c/is5qoa3V/2907-express-add-airbnb-coding-style)

File: .eslint-bin/js-list.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[
+  ".eslint-bin/lint-regitered-files.js",
+  ".eslint-bin/lint-regitered-files.js -> .eslint-bin/lint-registered-files.js",
+  ".eslint-bin/pre-commit-hook.js"
+]
```
Comment (arnaudbesnier) [inline .eslint-bin/js-list.json]: missing new line ;)

File: .eslint-bin/js-list.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[
+  ".eslint-bin/lint-regitered-files.js",
+  ".eslint-bin/lint-regitered-files.js -> .eslint-bin/lint-registered-files.js",
```
Comment (arnaudbesnier) [inline .eslint-bin/js-list.json]: Nope ;)

**Pull request merged**